scapy/contrib/isotp.py

```diff
@@ -57,289 +54,1346 @@ def answers(self, other):
             return self.payload.answers(other.payload)
         return 0
 
+    def __init__(self, *args, **kwargs):
```

Comment:
How about using

`def __init__(self, src=None, dst=None, exsrc=None, exdst=None, *args, **kwargs):`

instead, so that you don't have to use kwargs ?

Reply:
Ok got it